amivapi/settings.py

```diff
@@ -116,3 +116,37 @@
     'mtec',
     'gess'
 ]
+
+# Config for swagger (/docs)
+# TODO: should be linked to the setup.py information in the package
+SWAGGER_INFO = {
+    'title': 'AMIVAPI',
+    'version': '0.1-dev',
```

Comment:
I discussed this with @NotSpecial

It basically is the same information as in the package-info. However, we did find bith directions of import weird, so we left it like this for the moment.

Do you have a suggestion how to do it?